### PR TITLE
Feature/mat 3559

### PR DESCRIPTION
**Pull request by RohitKandimalla**

### CODE REVIEW

File: .github/workflows/gitleaks.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ jobs:
   gitleaks_scan:
     runs-on: ubuntu-latest
     env:
-      REPO: https://github.com/MeasureAuthoringTool/madie-layout
-      REMOTE_EXCLUDES_URL: https://raw.githubusercontent.com/semanticbits/bmat-gitleaks-automation/master/madie-layout/gitleaks.toml
+      REPO: https://github.com/MeasureAuthoringTool/madie-patient
+      REMOTE_EXCLUDES_URL: https://raw.githubusercontent.com/semanticbits/bmat-gitleaks-automation/master/madie-patient/gitleaks.toml
```
Comment (adongare) [inline .github/workflows/gitleaks.yml:10]: don't see `gitleaks.toml` file anywhere. is this something Ben adds or configured somewhere?

Comment (RohitKandimalla) [reply .github/workflows/gitleaks.yml:10]: I am trying to figure it out the same.

Comment (sb-benohe) [reply .github/workflows/gitleaks.yml:10]: I will have to create one, Would you like it to be cloned from a specific repo we had previously or would you like me to have a fresh config for this? @RohitKandimalla @adongare @gregory-akins

Comment (adongare) [reply .github/workflows/gitleaks.yml:10]: I'm okay with the way we are doing it for other micro frontend

Comment (sb-benohe) [reply .github/workflows/gitleaks.yml:10]: Fixed

Comment (adongare) [reply .github/workflows/gitleaks.yml:10]: Thanks @sb-benohe!

**Review by adongare — APPROVED** [review]

**Pull request merged**